uint8 seq
float64 gimbal_base_pitch
float64 gimbal_map_pitch